frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -720,9 +720,8 @@ export default function InteractiveWaveformModal({ selectedJob, setOpenInteracti
                 if (duplicatesPresent) {
                   setModalLabels(constantModalLabels.duplicate);
                   setModalOpen("duplicate");
-                } else {
-                  postNewJob();
                 }
+                postNewJob();
```

Comment:
I think they should be given the option to close the modal and make changes before confirming the job submission. I wouldn't submit the job until the modal has been handled

Reply:
If we submit the job here and as this modal appears, the job might become successful and another modal auto appears when a job is submitted so it would close this modal. It might be confusing. SO I would make this have 'Close' and 'Continue' buttons. If 'Continue', submit job, else if 'Close' then close modal and let the user keep editing